crates/core_arch/src/wasm32/simd128.rs

```diff
@@ -33,6 +33,7 @@ types! {
     /// type in WebAssembly. Operations on `v128` can only be performed with the
     /// functions in this module.
     // N.B., internals here are arbitrary.
+    #[target_feature(enable = "simd128")]
```

Comment:
Right, this doesn't work on types.
I guess we also don't want to add `cfg(target_feature)` here because otherwise using it would require `build-std`.